test/recipes/30-test_evp_data/evpkdf.txt

```diff
@@ -294,12 +294,12 @@ Ctrl.r = r:8
 Ctrl.p = p:1
 Output = 7023bdcb3afd7348461c06cd81fd38ebfda8fbba904f8e3ea9b543f6545da1f2d5432955613f0fcf62d49705242a9af9e61e85dc0d651e40dfcf017b45575887
 
-# Out of memory
+# Out of memory - request > 2 GB of memory
 KDF = scrypt
 Ctrl.pass = pass:pleaseletmein
 Ctrl.salt = salt:SodiumChloride
-Ctrl.N = N:1048576
+Ctrl.N = N:2097152
```

Comment:
Not sure I understand what this is fixing. Why is 2GB more correct than 1GB?

Reply:
Scrypt has a maximum that, by default, allows a bit over 1GB.  By bumping this up, it is guaranteed to be exceeded.  This means that the test is doing what it claims instead of failing with a bad key.